.gitignore

```diff
@@ -1,3 +1,4 @@
 log/
 .env
 *.sublime-project
+**/.DS_Store
```

Comment:
Traditionally system files should be ignored using a global .gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore